android/cli/commands/_build.js

```diff
@@ -4104,6 +4066,10 @@ AndroidBuilder.prototype.compileJavaClasses = function compileJavaClasses(next)
 
 	classpath[path.join(this.platformPath, 'lib', 'titanium-verify.jar')] = 1;
```

Comment:
do we need titanium-verify anymore?

Reply:
titanium-verify is responsible for validation Titanium modules, so yeah, I think we still need it.

Reply:
We definitely need it. When you include at least 1 module, we make a method call to this library on app startup to verify module licenses. Our mocha test suite is already indirectly testing this since it includes the "ti.map" module.